modules/virt-about-libguestfs-tools-virtctl-guestfs.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-libguestfs-tools-virtctl-guestfs.adoc
+
+[id="virt-about-libguestfs-tools-virtctl-guestfs_{context}"]
+= About libguestfs tools and virtctl guestfs
+
+The `virtctl guestfs` command contains many useful sub-commands that are helpful for modifying, inspecting, and debugging VM disks on a PVC. To see a complete list of possible `virt` commands, run `virt-` from a command line terminal. Some examples are:
+
+* To edit a file interactively in your terminal, use `virt-edit -a /dev/vda /etc/motd`.
+* To inject an ssh key into the guest and create a login, use `virt-customize -a /dev/vda --ssh-inject root:string:<public key example>`.
+* To see how much disk space is used by a VM, use `virt-df -a /dev/vda -h`.
+* To see the full list of all RPMs installed on a guest, use `virt-customize -a /dev/vda --run-command 'rpm -qa > /rpm-list'` to create an output file containing the full list. Then use `virt-cat -a /dev/vda /rpm-list` to display the output list in your terminal.
```

Comment:
I am happy to see these examples.

Reply:
Let's introduce the virtctl guestfs part first, then we will mention how the user can access the disk once the libguestfs-tools pod has been deployed.

Reply:
we need to mention that to doc: `virtctl guestfs` should not be used on running VMs and a VM should not be started if there is an active `virtctl guestfs` session.

Reply:
* To seal a virtual machine disk image to be used as a template, use `virt-sysprep -a /dev/vda`.